lib/evmone/execution_state.hpp

```diff
@@ -177,9 +192,9 @@ struct ExecutionState
     } analysis{};
 
     Stack stack;
+    StackSpace stack_space;
```

Comment:
Is this one kept last member for alignment reasons? Perhaps add a comment about it.